appshell/client_handler_mac.mm

```diff
@@ -61,6 +61,8 @@
   if ([window styleMask] & NSFullScreenWindowMask) {
       [window setTitle:str];
   }
+    
+  [NSApp changeWindowsItem:window title:str filename:NO];
```

Comment:
This to make sure we have a menu entry under Windows menu. As we are now nuking the window title, the menu entry is not getting added under 'Windows' menu. So I had to explicitly call this to make sure we have a menu entry. 

Please refer to [this](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/ApplicationKit/Classes/NSApplication_Class/#//apple_ref/occ/instm/NSApplication/changeWindowsItem:title:filename:
) link for documentation